### DISCUSSION TITLE
Init command

### DISCUSSION THREAD

**Discussion by shyim** [PHP-CS-Fixer/PHP-CS-Fixer#7940] [Feature ideas]
## Feature request

I am lazy as ..., everytime and creating the php-cs-fixer config manually IS SOOOOOOOOO annoying. I would like to have a simple command `php-cs-fixer init` which asks me which sets should be included and then creates config file and justs .gitignore maybe

**Comment by kubawerlos** [comment]
That sounds like a reasonable idea, do not hesitate to raise PR with it.

**Comment by shyim** [comment]
@Wirone maybe then delete feature request template? https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/.github/ISSUE_TEMPLATE/feature_request.md when discussion is preferred

**Reply by Wirone** [reply]
It's something that I did not manage to clean up fully. It shouldn't be deleted, but we need better descriptions. Now it should be easier, but I still need to find time for it.

**Reply by Wirone** [reply]
@shyim I've [prepared required changes](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7942), soon it will be more friendly and more clear 🙂.

**Comment by Wirone** [comment]
I've moved it to discussions, because I want us to have only actionable issues in the backlog, with clear definition of done. In general I like the idea, but let's discuss how should it work, and what is the scope of the command:

- should it just copy some dist file to project's root, with default settings, so it can be easily extended
- should it be interactive
- should it just focus on config file or also do other stuff (like Git ignore file)

When we have clear vision, we can make an issue with final requirement.
